src/Symfony/Component/Workflow/DataCollector/WorkflowDataCollector.php

```diff
@@ -24,6 +30,8 @@ final class WorkflowDataCollector extends DataCollector implements LateDataColle
 {
     public function __construct(
         private readonly iterable $workflows,
+        private readonly EventDispatcherInterface $eventDispatcher,
+        private FileLinkFormatter $fileLinkFormatter,
```

Comment:
those should be optional arguments to preserve BC

Reply:
This class has never been released :)